cfgov/jinja2/v1/sublanding-page/index.html

```diff
@@ -24,14 +24,9 @@
 
 {% block content_main %}
     {% for block in page.content -%}
-        {% if 'text_introduction' not in block.block_type
-           and 'hero' not in block.block_type %}
-            {% if 'heading' in block.block_type %}
-                <h2>{{ block }}</h2>
-            {% else %}
-                {{ render_stream_child(block) }}
-            {% endif %}
-        {% endif %}
+        <div class="block">
+            {{ render_stream_child(block) }}
+        </div>
```

Comment:
We discussed in scrum that it'll be up to content creators to ensure they don't add both, and editors to review the posts before publishing to catch any slip ups.